pylibfuzzer/exec/runner.py

```diff
@@ -119,12 +122,22 @@ def read_config(conf):
         # |SEED FILES|
         self.seed_files = config.get('seed_files')
 
+        if self.create_dataset:
+            self.fid = 0  # identifier for created datasets
+            self.dir_name = "dataset/dataset_" + str(datetime.now())
+            os.makedirs(self.dir_name, exists_ok=True)
+            logger.info("Creating dataset at %s", self.dir_name)
+
     def run(self):
         # execute the main loop
         self.input_generator.load_seed(self.seed_files)
-
         with self.dispatcher as cmd, self.input_generator, self.summarywriter.as_default():
             while not (self.input_generator.done() or self.timeout or self.overiter):
+                # prepare dataset
+                if self.create_dataset:
+                    dataset_in = open(self.dir_name + "/input_" + str(self.fid) + ".txt", 'wb')
+                    dataset_cov = open(self.dir_name + "/cov_" + str(self.fid) + ".txt", 'wb')
```

Comment:
please use `Path`instead of string concatenation. this implementation is not OS agnostic

Reply:
even better make the `self.dir_name` of type `Path`, so you only have to convert it once